### ISSUE TITLE
Identical sequences of random tags all pick the same choice

### ISSUE THREAD

**Issue by kirsle** [aichaos/rivescript-python#53]
If you have multiple `{random}...{/random}` sequences and they all look 100% identical (same words in the same order, etc.), then only one of them is actually processed (random choice chosen) and the random choice replaces _all_ occurrences of the sequence.

Example:

```
- {random}red|blue|green|black|white|gray|pink{/random} {random}red|blue|green|black|white|gray|pink{/random} {random}red|blue|green|black|white|gray|pink{/random}
```

This might yield an answer of "green green green".

The problem is likely because: after the random choice is picked, it does a regexp find/replace on the full original text of the tag with the answer, but it probably does a global replace, also replacing all other matching tag sequences. If you alternate the words in each set the problem is resolved.

**Comment by Dinh-Hung-Tu** [comment]
What is the file name of the code that handles the `{random}...{/random}` tag?

**Comment by kirsle** [comment]
process_tags() in brain.py

**Comment by Dinh-Hung-Tu** [comment]
@kirsle  Could you close this bug issue as well?

**Comment by kirsle** [comment]
Sure. Also a neat tip for the future: if you include text like "Fixes #53" in a pull request it will automatically close the mentioned issue when the PR is merged.
